shared/spotify-api/src/mockClient.ts

```diff
@@ -328,15 +353,26 @@ export class MockSpotifyClient implements SpotifyDataSource {
     limit?: number;
     offset?: number;
   }): Promise<Spotify.Object.Paginated<Spotify.Object.SavedAlbum>> {
-    throw new Error('Mock not implemented.');
+    return {
+      href: '',
+      offset: 0,
+      previous: '',
+      limit: 10,
+      next: '',
+      total: Object.keys(mocks.albums).length,
+      items: Object.values(mocks.albums).map((album) => ({
+        album,
+        added_at: '',
```

Comment:
Might be nice to actually provide a valid date string here. Perhaps `1970-01-01T00:00:00.000Z` which is the start of unix epoch?